dpdbaltics.php

```diff
@@ -96,7 +96,7 @@ public function __construct()
         $this->displayName = $this->l('DPDBaltics');
         $this->author = 'Invertus';
         $this->tab = 'shipping_logistics';
-        $this->version = '3.2.9';
+        $this->version = '3.3.0';
```

Comment:
why 3.3.0 and not 3.2.10? third number means small bug fix, second number means new feature and first one means major change.